vsintegration/src/FSharp.Editor/FSharpKeywords.fs

```diff
@@ -0,0 +1,90 @@
+// Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+module Microsoft.VisualStudio.FSharp.Editor.FSharpKeywords
+  
+let keywordsWithDescription : (string * string) list =
+    [ "abstract",  """Indicates a method that either has no implementation in the type in which it is declared or that is virtual and has a default implementation."""
```

Comment:
I'm a bit puzzled about how this stuff works. What `FSComp.txt` contains exactly? Some lines contain 2 comma delimited parts, others - 3. Some start with a number, some - with a word... 
`lexhelp.fs` already contains keyword list https://github.com/Microsoft/visualfsharp/blob/master/src/fsharp/lexhelp.fs#L199, but it does not include some, like `let!`. 
Why not just move this `keywordsWithDescription` to `lexhelp.fs`?